zebra-scan/tests/scan_task_commands.rs

```diff
@@ -5,12 +5,14 @@
 //!
 //! export ZEBRA_CACHED_STATE_DIR="/path/to/zebra/state"
 //! cargo test scan_task_commands --features="shielded-scan" -- --ignored --nocapture
+#![allow(dead_code)]
```

Comment:
It's strange that this is necessary, since all of the code here is used, but it seems fine for now.